### PR TITLE
[Doc Improvement] [Update configure your app in Azure AD][3172092]

### PR DESCRIPTION
**Pull request by v-bvishnu**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/authentication/bot-sso-register-aad.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4801737d96ec6c66746f3f1e887f602a452d13a1/msteams-platform%2Fbots%2Fhow-to%2Fauthentication%2Fbot-sso-register-aad.md) | 93 | 34 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9d8d6dff-919e-4af5-9ad4-47178c115483) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/bots/how-to/authentication/bot-sso-register-aad.md [hunk 1]
```diff
@@ -844,6 +844,16 @@ Congratulations! You've completed the following app configurations in Azure AD r
 
 ---
 
+## Best practices
+
+- Keep the Azure AD app registration restricted to its original purpose of service to service application.
```
Comment (blackchoey) [inline msteams-platform/bots/how-to/authentication/bot-sso-register-aad.md:849]: minor: I'm not sure whether our customer will be confused about what does the "Azure AD app registration" refers to. Do you think we need to add some description to it like `the Azure AD app registration created for Azure Bot service`?

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/authentication/bot-sso-register-aad.md](https://github.com/MicrosoftDocs/msteams-docs/blob/480b79752309860b5ab970136b95e92b730d4afc/msteams-platform%2Fbots%2Fhow-to%2Fauthentication%2Fbot-sso-register-aad.md) | 94 | 29 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/19fedbfd-2239-4586-a7d6-160380991b1a) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-gs-workflow-bot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/480b79752309860b5ab970136b95e92b730d4afc/msteams-platform%2Fsbs-gs-workflow-bot.yml) | 89 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ef330bd5-2825-4b78-b28d-0ea3aa924a25) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/authentication/bot-sso-register-aad.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0c88bf0cce09fa0d70ce0814ab95ee0185c5205a/msteams-platform%2Fbots%2Fhow-to%2Fauthentication%2Fbot-sso-register-aad.md) | 94 | 29 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1c2b7dcc-0258-46a3-8716-e9c751a4b303) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-gs-workflow-bot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/0c88bf0cce09fa0d70ce0814ab95ee0185c5205a/msteams-platform%2Fsbs-gs-workflow-bot.yml) | 89 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d31864ae-2fad-4529-babe-f7153c831791) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/authentication/bot-sso-register-aad.md](https://github.com/MicrosoftDocs/msteams-docs/blob/70976e41f4f7876e5ec58e23672db2d75c5bd659/msteams-platform%2Fbots%2Fhow-to%2Fauthentication%2Fbot-sso-register-aad.md) | 94 | 29 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6d47cfc8-5d1c-4343-8953-5c4b52b74342) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-gs-workflow-bot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/70976e41f4f7876e5ec58e23672db2d75c5bd659/msteams-platform%2Fsbs-gs-workflow-bot.yml) | 89 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/069d2a7c-0026-4746-87c3-7b0fabe248d1) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/authentication/bot-sso-register-aad.md](https://github.com/MicrosoftDocs/msteams-docs/blob/df436ad670f89aa5a9dea40273518650a9a36048/msteams-platform%2Fbots%2Fhow-to%2Fauthentication%2Fbot-sso-register-aad.md) | 94 | 29 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a646563b-9599-4d38-a0ed-0c69911d1496) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-gs-workflow-bot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/df436ad670f89aa5a9dea40273518650a9a36048/msteams-platform%2Fsbs-gs-workflow-bot.yml) | 89 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/216f67b4-742d-4b08-9c1c-4071c5561bb2) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/authentication/bot-sso-register-aad.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4a4c9d011c1eec88eca8a9bb5b522997b467d7a2/msteams-platform%2Fbots%2Fhow-to%2Fauthentication%2Fbot-sso-register-aad.md) | 94 | 29 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/29688bbe-f0ce-4aec-852d-4ce6a1b820af) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-gs-workflow-bot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/4a4c9d011c1eec88eca8a9bb5b522997b467d7a2/msteams-platform%2Fsbs-gs-workflow-bot.yml) | 89 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5c8e4273-156c-4204-8625-582d4bef1597) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/authentication/bot-sso-register-aad.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4d04ff39b8c434dc69f62eaad4ee3aaee4636037/msteams-platform%2Fbots%2Fhow-to%2Fauthentication%2Fbot-sso-register-aad.md) | 94 | 30 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/469674f5-d1d0-4bd7-a5dd-23961c07874f) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-gs-workflow-bot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/4d04ff39b8c434dc69f62eaad4ee3aaee4636037/msteams-platform%2Fsbs-gs-workflow-bot.yml) | 89 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1eaf589a-e201-4cb2-849f-4756874ef187) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by blackchoey — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/authentication/bot-sso-register-aad.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e2f73f7f825c3648a10f4b34b88608e1e2b8f100/msteams-platform%2Fbots%2Fhow-to%2Fauthentication%2Fbot-sso-register-aad.md) | 94 | 30 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8b1a5639-580c-43fd-9a94-b929b3072e6d) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-gs-workflow-bot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/e2f73f7f825c3648a10f4b34b88608e1e2b8f100/msteams-platform%2Fsbs-gs-workflow-bot.yml) | 89 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1048686c-df59-4b95-aaec-3891cdbdf174) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-ypalikila — APPROVED** [review]
Approved.

**Pull request merged**